lua/treesitter-context/context.lua

```diff
@@ -80,9 +80,9 @@ end
 --- context node.
 --- @param node TSNode
 --- @param query vim.treesitter.Query
+--- @param bufnr integer
 --- @return Range4?
-local context_range = cache.memoize(function(node, query)
-  local bufnr = api.nvim_get_current_buf()
+local context_range = cache.memoize(function(node, query, bufnr)
```

Comment:
I think `hash_node()` should also be updated so it takes into account the buffer number.